source/guide/3.2/getting-started-bootstrapping.md

```diff
@@ -38,12 +38,10 @@ Clone the [Cloudify-Manager-Blueprints](https://github.com/cloudify-cosmo/cloudi
 mkdir -p ~/cloudify-manager
 cd ~/cloudify-manager
 git clone https://github.com/cloudify-cosmo/cloudify-manager-blueprints
+cd cloudify-manager-blueprints
+git checkout -b <tag> <tag>
 {% endhighlight %}
 
-{%note title=Note%}
-Make sure you use a tag from [Cloudify-Manager-Blueprints](https://github.com/cloudify-cosmo/cloudify-manager-blueprints/releases) that matches your Cloudify version. Blueprints taken from the master branch might not work for you.
```

Comment:
I think that while it's good to add the two commands above.. this is still an important note.